packages/playground/windows/playground/MainPage.idl

```diff
@@ -1,10 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+#include "../../../../vNext/include/NamespaceRedirect.h"
```

Comment:
>../../../../vNext/include/ [](start = 10, length = 26)

Should we add this folder as `Midl.AdditionalIncludeDirectories` in the vcxproj file so we don't have to '..' our way out of the folder?

Reply:
At least when we go to ship these headers as a nuget package, thenuget should add the additional includes and this code won't need to be updated...

---
In reply to: [419713227](https://github.com/microsoft/react-native-windows/pull/4766#discussion_r419713227) [](ancestors = 419713227)